### DISCUSSION TITLE
Collapse All button in the Org Browser

### DISCUSSION THREAD

**Discussion by dannybusiness** [forcedotcom/salesforcedx-vscode#5794] [Ideas]
When I am navigating the org browser looking for some metadata to retrieve to my local environment, I often get lost in the length of the list of items as I expand more and more metadata types. To solve this, I would love to see a 'collapse all' button like exists on the top of the file explorer to allow me to collapse all metadata types and effectively start my browsing from a cleaner list of metadata types.

**Comment by smaddox-sf** [comment]
Thanks for the feedback @dannybusiness !  We'll consider this as we do work on Org Browser

**Comment by mshanemc** [comment]
done!
